install/installer/cmd/mirror_list.go

```diff
@@ -85,6 +86,117 @@ func init() {
 	mirrorListCmd.Flags().StringVarP(&mirrorListOpts.ConfigFN, "config", "c", os.Getenv("GITPOD_INSTALLER_CONFIG"), "path to the config file")
 }
 
+func renderAllKubernetesObject(cfgVersion string, cfg *configv1.Config) ([]string, error) {
```

Comment:
If I understand this function correctly it tries to exercise all code paths which might ever require an image. How do we make sure we don't inadvertently introduce a code path down the road which is not covered here, and end up with a broken airgapped installer?